src/main/java/com/karol/kindergartenmanagementsystem/config/SeedAdminAccountConfiguration.java

```diff
@@ -0,0 +1,41 @@
+package com.karol.kindergartenmanagementsystem.config;
+
+import com.karol.kindergartenmanagementsystem.model.Role;
+import com.karol.kindergartenmanagementsystem.model.User;
+import com.karol.kindergartenmanagementsystem.repository.UserRepository;
+import com.karol.kindergartenmanagementsystem.service.UserAccountDetailsService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+@Slf4j
+public class SeedAdminAccountConfiguration implements CommandLineRunner {
```

Comment:
I would stick to one way of declaring beans  - by @Bean in configuration class or @Component . (I prefer @Bean, because it easier to keep them in one place).

// It doesn't apply to specified annotatnios like f.e. repository or handler